extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -277,4 +289,16 @@ private static function linkifyMediaCaption( Parser $parser, &$media ) {
 			$parser->replaceLinkHolders( $media['caption'] );
 		}
 	}
+
+	/**
+	 * @param $handlerParams
+	 * @desc Determines if height or width of image was set
+	 * explicitly by user by adding '{width}px' or 'x{height}px' to image wikitext
+	 * @return bool true if the image size was manipulated by user
+	*/
+	private static function isFixedImage( $handlerParams ) {
```

Comment:
Yup, this method actually means something along the lines of isIcon or isFormattedAsIcon